macaw-aarch32/tests/arm/Makefile

```diff
@@ -28,6 +28,10 @@ all: $(patsubst %.c,%-a32.exe,$(wildcard *.c)) $(patsubst %.c,%-t32.exe,$(wildca
 %-mixed.s: %-mixed.c
 	$(A32CC) -fno-stack-protector -S -c $< -o $@
 
+# doesn't work witn arm-none-eabi
+test-empty-section.exe: test-empty-section.c
```

Comment:
This `Makefile` says `test-empty-section.exe`, but the checked-in file is named `test-empty-section-a32.exe`. Which is it?

Reply:
Good catch, it should have -a32 appended